installer/common/shared.sh

```diff
@@ -135,9 +136,6 @@ elif [ "${KAFKA}" = "ExternalStrimzi" ]; then
         print_error_message "Strimzi topic operator configuration is invalid."
         print_error_message "You must define environment variable 'KAFKA_STRIMZI_TOPIC_OPERATOR_NAMESPACE' and set it to a namespace that contains a running strimzi-topic-operator pod."
         print_error_message ""
-        ##TODO FIX_PIPELINES_REF
-        print_error_message "More information can be found here: https://developer.lightbend.com/docs/pipelines/current/index.html#_existing_strimzi_kafka"
-        print_error_message ""
```

Comment:
@lightbend/cloudflow-team I'm removing these URLs for now because they are no longer reachable. Let me know if you think there are alternative good links that I can put here instead.

Reply:
We have a backlog item to port those old documentation pages to the new docs. We are also planning some major installer updates so for now I think this is acceptable